sources/packages/api/src/route-controllers/application/application.controller.service.ts

```diff
@@ -203,4 +209,76 @@ export class ApplicationControllerService {
       );
     }
   }
+
+  /**
+   * Process application income verification details.
+   * @param CRAIncomeVerification CRA income verification data.
+   * @return processed object.
+   */
+  processApplicationIncomeVerificationDetails(
+    CRAIncomeVerification: CRAIncomeVerification[],
+  ): ApplicationIncomeVerification {
+    const incomeVerificationDetails = {} as ApplicationIncomeVerification;
+    const studentIncomeVerifications = CRAIncomeVerification.filter(
```

Comment:
studentIncomeVerification can be assigned directly like this.

const [studentIncomeVerification] = CRAIncomeVerification.filter(
      (incomeVerification) => !incomeVerification.supportingUser,
    );